campaignion_tracking/campaignion_tracking_gtm/campaignion_tracking_gtm.module

```diff
@@ -42,6 +42,6 @@ function campaignion_tracking_gtm_campaignion_tracking_snippets() {
 function campaignion_tracking_gtm_d7csp_hosts() {
   $hosts['connect-src'][] = 'https://www.google.com';
   $hosts['frame-src'][] = 'https://www.googletagmanager.com';
-  $hosts['img-src'][] = 'https://www.google-analytics.com';
+  $hosts['img-src'][] = 'https://www.google-analytics.com https://www.googletagmanager.com';
```

Comment:
```suggestion
  $hosts['img-src'][] = 'https://www.google-analytics.com';
  $hosts['img-src'][] = 'https://www.googletagmanager.com';
```

Reply:
Sure, I can do that! Should I simply commit your suggestion via the fancy (for me new) interface option? :) Also, is that how it should look like generally? I did it in the same way for some profile updates, should I separate them out as well? (e.g. https://gitlab.more-onion.com/drupal/actmo/-/merge_requests/20/diffs)

Reply:
Yes, please.

Reply:
I'll assume this is a computer scientist yes please as an answer to both questions and will go ahead ;)